src/components/AddBook.js

```diff
@@ -1,21 +1,33 @@
 import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
 import { v4 as uuidv4 } from 'uuid';
+import Endpoints from '../Endpoints';
 import { addBook } from '../redux/books/books';
 
 export default function AddBook() {
   const dispatch = useDispatch();
-  const [title, setTitle] = useState('');
+  const [bookTitle, setTitle] = useState('');
   const submitBookToStore = (e) => {
     e.preventDefault();
 
     const newBook = {
-      id: uuidv4(),
-      title,
+      item_id: uuidv4(),
+      title: bookTitle,
       category: 'Random',
       author: 'Jeff',
     };
```

Comment:
- [ ] (Optional) It would be nice to input a dynamic value for the `author` and `category` of your `newBook` object. You can implement this feature the same way you implemented the `title` by using React's `useState` hook to dynamically set the value to the one inputted by the user of the app. The essence of the app is that a user can input their own values for `title`, `author`, and choose a `category` from a list of categories provided in a `select` form control.

Reply:
Hi @Gambit142 ,

Thanks for the review. Firstly, I'll remove the commented-out code now before I even merge it- thanks. Secondly, I didn't make the `author` and `category` dynamic because the page itself is limited to 1 input field for the title and not the autor. Then for the category, it wasn't required, so instead of having it empty, I though I could put some placeholders to hold the structure. Otherwise, I really appreciate your review.